docs/content/users/usage/commands.md

```diff
@@ -539,9 +539,9 @@ Dump a database to a file or to stdout.
 Flags:
 
 * `--bzip2`: Use bzip2 compression.
-* `--file`, `-f`: Provide the path to output the dump.
-* `--gzip`, `-z`: Use gzip compression. (default `true`)
-* `--target-db`, `-d`: If provided, target-db is alternate database to export. (default `"db"`)
+* `--database`, `-d`: Target database to export from (default "db")
+* `--file`, `-f`: Path to a SQL dump file to export to
+* `--gzip`: Use gzip compression (default true)
```

Comment:
I’d keep the backticks to represent literal values, unless there’s a good reason to drop them:

```suggestion
* `--database`, `-d`: Target database to export from (default `"db"`)
* `--file`, `-f`: Path to a SQL dump file to export to
* `--gzip`: Use gzip compression (default `true`)
```

Did you remove the `-z` alias on purpose?

Reply:
Yes, the -z flag is deprecated now. Backticks were removed by accident, I just copied the new texts from the command line output and there of course are none. And thanks for your review!

Reply:
How did `-z` get deprecated? :)

Reply:
Well, I deprecate it with this PR see https://github.com/ddev/ddev/pull/5013/files#diff-a08f845a12c4812bdf8f9649f2e8548658439231e8ca9d39038ea03be3f5c70fR97. Imho -z is a very bad name the flag does not need a shorthand in the first place, the other compression flags don't have shorthands either. But if you don't agree I can let it stay like it is.

Reply:
Please discuss deprecations rather than just doing them. If there were a discussion in a tracked location, that would be something, but just deprecating because you think it should deprecate is different. 

Are you still supporting `-z` here? It's the default anyway, so maybe people don't use it. But if people have scripts that use it we'll break them, and since it's unlikely to be used anyway... why bother deprecating?

Reply:
Sure, it still works. I did not remove but deprecate it because there is no value and it's unclear in meaning as explained above. If you don't agree, we can remove the deprecation as well.

I did not touch any test which makes sure nothing is breaking here.

Reply:
It's so useless anyway since it's the default, so no need to change anything at this point. Let's just have agreement about deprecations before doing them. (I'm not sure there's a test that would use -z)

Reply:
There's no risk -z not working anymore, just marked as deprecated and if --gzip works -z works too.

Reply:
Done